Spatial.tscn

```diff
@@ -71,28 +71,27 @@ scale = Vector2( 2, 2 )
 [node name="Line2D" parent="Control/MiniMap" instance=ExtResource( 4 )]
 
 [node name="GlobalMenuButton" type="Control" parent="Control"]
-margin_left = 287.348
-margin_right = 314.348
-margin_bottom = 32.0
+margin_left = 321.348
+margin_top = 4.0
```

Comment:
You took this off of the texture, now it does not line up to the top of the screen.
![Screenshot from 2023-08-24 21-12-58](https://github.com/AsherGlick/Burrito/assets/603989/7527bd8f-90e7-49e5-9f75-0e25c0a507dc)